### PR TITLE
Add issues templates

### PR DESCRIPTION
**Pull request by piotr-oles**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+assignees: ''
+
+---
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+## Expected behavior
+<!-- Describe what the desired behavior would be. -->
+
```
Comment (johnnyreilly) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: Maybe also add something like this?:

```
### Steps to Reproduce the Problem

### Location of a Minimal Repository that Demonstrates the Issue.
```

I'd say 90% of reported issues don't provide that and it means no-one can really look into them.

**Review by johnnyreilly — CHANGES_REQUESTED** [review]
Great start! I've a suggested addition

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (piotr-oles) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: Done :)

**Review by johnnyreilly — APPROVED** [review]

**Comment by johnnyreilly** [comment]
Nice one! Let's merge this ❤️😊

**Comment by piotr-oles** [comment]
Travis CI got stuck on OSX builds :/ I hope it will recover somehow.

**Comment by johnnyreilly** [comment]
It'll get there in the end! (Mac builds don't parallelise on Travis and so builds take a while now)

**Pull request merged**